### PR TITLE
[docs][autoscaler] Autoscaler docs and default config improvement

### PR DESCRIPTION
**Pull request by gyfora**
## What is the purpose of the change

Improve the autoscaler documentation based on the original release blogpost and some recent changes.

## Brief change log

- Changed default catch up duration to 15m to reduce unexpected large scale ups

## Verifying this change

Not applicable

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changes to the `CustomResourceDescriptors`: no
  - Core observer or reconciler logic that is regularly executed: no

## Documentation

  - Does this pull request introduce a new feature? no

### CODE REVIEW

**Comment by gyfora** [comment]
cc @1996fanrui

File: docs/content/docs/custom-resource/autoscaler.md [hunk 1]
```diff
@@ -35,26 +35,78 @@ Key benefits to the user:
...
+
+```
+jobmanager.scheduler: adaptive
+```
+
+## Job Requirements and Limitations
+
+### Requirements
+
+The autoscaler currently only works with the latest [Flink 1.17](https://hub.docker.com/_/flink) or after backporting the following fixes to your 1.15/1.16 Flink image:
```
Comment (1996fanrui) [inline docs/content/docs/custom-resource/autoscaler.md:90]:
```suggestion
The autoscaler currently only works with [Flink 1.17](https://hub.docker.com/_/flink) and later flink images, or after backporting the following fixes to your 1.15/1.16 Flink images:
```

How about updating the `latest Flink 1.17` to `Flink 1.17 and later flink images`? The latest Flink 1.17 is not clear, it means the latest 1.17.1 is supported, but 1.17.0 is not supported. The fact is that 1.17 and all versions after that support it.

**Review by 1996fanrui — COMMENTED** [review]
Hi @gyfora , thanks for the update. Overall is good to me, I just left a minor comment, please take a look in your free time, thanks~

File: docs/content/docs/custom-resource/autoscaler.md (hunk 1 above)
Comment (gyfora) [reply docs/content/docs/custom-resource/autoscaler.md:90]: makes sense :)

**Pull request merged**